lib/node_modules/@stdlib/utils/parse-ndjson/package.json

```
@@ -0,0 +1,68 @@
+{
+    "name": "@stdlib/utils/parse-ndjson",
+    "version": "0.0.0",
+    "description": "Parse a string as Newline-Delimited JSON (NDJSON).",
```

```suggestion
    "description": "Parse a string as newline-delimited JSON (NDJSON).",
```

"newline" and "delimited" are not capitalized. This should be changed here and elsewhere.